builder/build.go

```diff
@@ -209,9 +209,46 @@ func createBuildTemplate(functionName string, handler string, language string, u
 		}
 	}
 
+	for _, extraPath := range copyExtraPaths {
+		extraPathAbs, err := pathInScope(extraPath, ".")
+		if err != nil {
+			return tempPath, err
+		}
+		copyErr := CopyFiles(
+			extraPathAbs,
+			filepath.Clean(tempPath+"/"+extraPath),
```

Comment:
I believe the "extra paths" should be copied into the `functionPath` to ensure they are at the same directory level as the rest of the function handler. This should allow using the extra files without any other changes to existing templates.

```suggestion
			filepath.Clean(functionPath+"/"+extraPath),
```

Reply:
The only potential issue with this is that there are much more likely to be file path collisions, you could have `data` in the top of the project and then `data` inside your function. The default behavior would result in a merge/overwrite of the existing `function/data`? 

If we all agree that this is what we want, then i am more than happy to change it.

Reply:
That is more what I expected and I saw the PR subject saying that it is copying common files into the function's handler folder.